### ISSUE TITLE
Typo: 'SIF' in lower case in main pom.xml

### ISSUE THREAD

**Issue by tdeenes** [Access4Learning/sif3-framework-java#36]
In the rows 236-238 of the root pom.xml file, modules are referred to as 'SIF3Common', 'SIF3InfraModel', 'SIF3InfraCommon'. However, those folders are named as 'sif3Common', 'sif3InfraModel', 'sif3InfraCommon' (note 'sif' instead of 'SIF'). 

```
<modules>
	<module>SIF3Common</module>
	<module>SIF3InfraModel</module>
	<module>SIF3InfraCommon</module>
	<module>SIF3REST</module>
</modules>
```

The above lines should be written as:
```
<modules>
	<module>sif3Common</module>
	<module>sif3InfraModel</module>
	<module>sif3InfraCommon</module>
	<module>SIF3REST</module>
</modules>
```

**Comment by joerghuber** [comment]
Thanks for picking that one up. I will fix this with the next release.

**Comment by joerghuber** [comment]
Fixed on nsip fork in v0.13.0. Will be pushed to this repository in the near future.

**Comment by joerghuber** [comment]
This is now merged to the master.
